examples/next/docs/react/demo/src/renderers/ProgressBar.tsx

```diff
@@ -1,24 +1,33 @@
-import React from "react";
-import { BaseEditorComponent, HotEditorProps } from '@handsontable/react';
+import React from 'react';
+import { BaseEditorComponent } from '@handsontable/react';
 import { getRangeValue } from './utils';
+import { MESSAGE } from '../constants';
 
 const minAllowedValue = 0;
 const maxAllowedValue = 100;
 
-
-interface IProgressBarRendererProps extends HotEditorProps {
-  value: number;
-}
-
 export class ProgressBarRenderer extends BaseEditorComponent {
```

Comment:
[In the first point of my earlier comment](https://github.com/handsontable/handsontable/pull/9935#issuecomment-1415532583), I suggested that extending a _renderer_ class from an _editor_ class is not needed in this example.
And if there wouldn't be a need for extending classes, we should probably use function-based components here.